src/psyclone/tests/gocean1p0_test.py

```diff
@@ -43,8 +43,9 @@
 import pytest
 from psyclone.configuration import Config
 from psyclone.parse.algorithm import parse
-from psyclone.psyGen import PSyFactory, InternalError
-from psyclone.generator import GenerationError, ParseError
+from psyclone.errors import InternalError, GenerationError
+from psyclone.psyGen import PSyFactory
+from psyclone.generator import ParseError
```

Comment:
Might be clearer to change this to:

    from psyclone.parse.utils import ParseError

Reply:
Done